st2common/st2common/util/config_loader.py

```diff
@@ -169,6 +169,11 @@ def _assign_dynamic_config_values(self, schema, config, parent_keys=None):
                 is_jinja_expression = jinja_utils.is_jinja_expression(
                     value=config_item_value
                 )
+                if "decrypt_kv" in str(config_item_value):
+                    LOG.audit("User %s is decrypting the value %s from the config within pack %s", self.user,
+                              config_item_value, self.pack_name,
+                    extra = {"user": self.user, "key_name": config_item_key, "pack_name": self.pack_name,
+                             "config_item_value": config_item_value, "operation": "pack_config_value_decrypt"})
```

Comment:
Let's make sure that Jinja parts are not in the logs, excluding `config_item_value`:

```suggestion
                if "decrypt_kv" in str(config_item_value):
                    LOG.audit("User %s is decrypting the value for key %s from the config within pack %s", self.user,
                              config_item_key, self.pack_name,
                              extra = {"user": self.user, "key_name": config_item_key, "pack_name": self.pack_name,
                                       "operation": "pack_config_value_decrypt"})
```

Reply:
![image](https://user-images.githubusercontent.com/40653159/160940273-c496a513-e703-4df3-932b-55876a9fa9a4.png)
30 22:17:18,193 AUDIT [-] User st2admin is decrypting the value for key f5_VIP_address from the config within pack cbanso_demo (user='st2admin',key_name='f5_VIP_address',pack_name='cbanso_demo',operation='pack_config_value_decrypt')

Hi @armab , 
Added your changes it now looks like the image above!